stormlog/jax/utils.py

```
@@ -165,9 +216,24 @@ def get_backend_info() -> Dict[str, Any]:
     Returns a dictionary with the JAX runtime backend classification
     and platform details.
     """
+    raw_backend = "cpu"
+    if JAX_AVAILABLE:
+        try:
+            raw_backend = str(jax.default_backend())
+        except Exception:
+            pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)